src/Compatibility/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -207,4 +207,7 @@
   <data name="TypeIsEffectivelySealed" xml:space="preserve">
     <value>Type '{0}' is sealed because it has no visible constructor on {1} but it does on {2}</value>
   </data>
-</root>
+  <data name="EnumValuesMustMatch" xml:space="preserve">
+    <value>Field '{0}.{1}' changed from '{2}' to '{3}'.</value>
```

Comment:
```suggestion
    <value>Field '{0}.{1}' changed its value from '{2}' to '{3}'.</value>
```
Perhaps we can be more specific on the error? I also see that we kind of have different style of error messages across the diagnostics, some seem to state a fact like "Type bla does not inherit from bla, but it did before", while others use a more explisive description like: "Cannot add interface member bla because it did not exist before". I wonder if we should try to unify them.

Reply:
Hmm yeah, I was trying to find some commonality across the other messages, but they were all different. I can file an issue about this, so that I can offer my suggestions on how to unify them.

Reply:
Filed https://github.com/dotnet/sdk/issues/25565.